app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Fantastic job on adding devise authentication to the Blog app. You have shown great dedication in following Rails conventions and writing clean and organised code to deliver an good project 👏🏾 💯 

**Project requirements:**

Kindly note that the for this milestone, it is a requirement to add email confirmation after user registration.

![image](https://github.com/Mike111222/BlogApp/assets/38088818/0edcd8ee-7c15-4de6-ad23-79cf94bb6723)


**Suggestion:***

- Email confirmation can be implemented by commenting the the `:confirmable` keyword on the use model. The `:confirmable` keyword in the Rails User model enables email confirmation during registration for additional security and handles automated email confirmation flow.


Overall, the Device authentication works as expected and with these few adjustments your project will be ready for approval. Great job!!! 🚀